the-list-be/server.js

```diff
@@ -10,22 +10,50 @@ const uri = 'mongodb://localhost:27017';
 const client = new MongoClient(uri);
 const db = client.db('database');
 const students = db.collection('students');
+const dailyAttendance = db.collection('dailyAttendance');
+
+router.patch('/attendance/:date', async ctx => {
+  if (ctx.request.body.action == 'add') {
+    await dailyAttendance.updateOne(
+      { date: ctx.params.date },
+      { $addToSet: { students: ctx.request.body._id } },
+      { upsert: true }
+    );
+  }
+  if (ctx.request.body.action == 'remove') {
```

Comment:
```suggestion
  } else if (ctx.request.body.action == 'remove') {
```

А что произойдёт, если я передам `"action": "random string"`? Надо, в этом случае, адекватную ошибку на клиент отправить.